### DISCUSSION TITLE
Question: best loss function for deep face recognition

### DISCUSSION THREAD

**Discussion by pfeatherstone** [KevinMusgrave/pytorch-metric-learning#662] [Q&A]
This library looks great and offers loads of loss functions suitable for face recognition.

Which loss do you think is state-of-the-art, easiest to train and provides best results on either VGGFace2, Casia-WebFace, or LFW for face recognition. Specifically, a model that outputs a rank-1 N-dimensional metric space embedding with L2 norm.

Many thanks

**Comment by KevinMusgrave** [comment]
Did you try ArcFace?
